### ISSUE TITLE
Your card was declined.

### ISSUE THREAD

**Issue by artalar** [CompuIves/codesandbox-client#1229]
# 🐛 bug report

## How has this issue affected you? What are you trying to accomplish?
Half year ago I was trying to pay the subscription (patron) but got an error and forgot about that for some time. Today I was trying again and receive the error `Your card was declined.`.
I have a friend with the same problem, we from Russia, сould that be the case?

> request `https://codesandbox.io/api/v1/users/current_user/subscription` date `date: Wed, 24 Oct 2018 08:46:01 GMT`

### Your Environment
| Software         | Name/Version|
| ---------------- | ---------- |
| Сodesandbox  | PROD-1540291027-be80aee
| Browser | Yandex 18.9.1.954
| Operating System | Windows 7

**Comment by goodmind** [comment]
@CompuIves

**Comment by candenzl** [comment]
I have the same problem and i'm from Russia too. Is it solved?

**Comment by CodeSpartan** [comment]
Same here, also from Russia

**Comment by lbogdan** [comment]
Closing this, as we're using [Stripe](https://stripe.com/) for credit card processing, and unfortunately we don't have any control over their card validation process.

**Comment by CompuIves** [comment]
Hey everyone, I wanted to give some context on this.

When we just released CodeSandbox Patrons our country of residence (The Netherlands) had an embargo on Russia. We weren't sure whether we were allowed to sell Patron subscriptions to people living in Russia and we also didn't have the money to ask a lawyer about this issue. We asked Stripe to decline cards from Russia at the time and whenever I got an e-mail from someone with a declined card I would give them free Patron. However, as CodeSandbox became more popular I started getting more and more messages about declined cards and I started slacking.

We've now looked into the embargo and it seems like it isn't in effect anymore. I've removed the block and it should now work as intended.

I'm sorry for the silence here, I didn't really have a good answer to the question but I should've at least sent a message about the current situation.
